monkey/infection_monkey/master/mock_master.py

```diff
@@ -101,20 +101,44 @@ def _fingerprint(self):
 
     def _exploit(self):
         logger.info("Exploiting victims")
-        result, info, attempts, error_message = self._puppet.exploit_host(
-            "PowerShellExploiter", "10.0.0.1", {}, None
-        )
+        (
+            exploitation_result,
+            propagation_result,
+            os,
+            info,
+            attempts,
+            error_message,
+        ) = self._puppet.exploit_host("PowerShellExploiter", "10.0.0.1", {}, None)
```

Comment:
Instead of unpacking this, let's modify `ExploitTelem` to accept an `ExploiterResultsData` object.